misc/discoverynode/README.md

```diff
@@ -2,11 +2,41 @@
 
 ## Notes
 
-`vendor` discovery family is a counter which increments every second
+`vendor` discovery family is actually sequential number generator at one second increments.
```

Comment:
For testing. At the moment, I'm forced to have some "vendor" discovery because that's what sequencer uses, and it makes testing a whole lot easier, e.g. run "vendor" discovery for ten seconds, should receive ten events [1,2,3,4,5,6,7,8,9,10], etc.